src/main/java/org/ballerinalang/command/cmd/RemoveCommand.java

```diff
@@ -106,36 +119,27 @@ private void remove(String version) {
         }
     }
 
-    /**
-     * Delete files inside directories.
-     *
-     * @param dirPath directory path
-     * @param outStream output stream
-     *      @param version deleting version
-     * @throws IOException throw an exception if an issue occurs
-     */
-    public static void deleteFiles(Path dirPath, PrintStream outStream, String version) throws IOException {
-        if (dirPath == null) {
-            return;
-        }
-
-        Files.walk(dirPath)
-                .sorted(Comparator.reverseOrder())
-                .forEach(path -> {
-                    if (!Files.isWritable(path)) {
-                        throw ErrorUtil.createCommandException("permission denied: you do not have write access to '" +
-                                dirPath + "'");
-                    }
-                });
-
-        Files.walk(dirPath)
-                .sorted(Comparator.reverseOrder())
-                .forEach(path -> {
-                    try {
-                        Files.delete(path);
-                    } catch (IOException e) {
-                        throw ErrorUtil.createCommandException("cannot remove '" + path + "'");
+    private void clean() {
+        try {
+            File folder = new File(ToolUtil.getDistributionsPath());
+            File[] listOfFiles;
+            listOfFiles = folder.listFiles();
+            if (listOfFiles.length == 2) {
+                getPrintStream().println("There is nothing to remove since active distribution only exists");
```

Comment:
Shall we change the message to "There is nothing to remove. Only active distribution is remaining."